pages/common/gradle.md

```diff
@@ -3,19 +3,19 @@
 > Gradle is an open source build automation system.
 > More information: <https://gradle.org>.
 
-- Compile a package:
+- Assemble outputs and run checks:
 
 `gradle build`
 
-- Exclude test task:
+- Exclude the `test` task:
 
 `gradle build -x {{test}}`
```

Comment:
Is there a reason, why `test` is used as an example? Generally I prefer more gerenic examples, like:
```suggestion
- Exclude a specific task:

`gradle build -x {{task_name}}`
```